io.github.alescdb.mailviewer.json

```diff
@@ -0,0 +1,67 @@
+{
+  "id": "io.github.alescdb.mailviewer",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command": "mailviewer",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--filesystem=host",
```

Comment:
why does this need `host` permissions?

Reply:
`host` was for my personal use case, I need to access eml files anywhere on my filesystem.
I've changed to `home`.